lib/backend/fetch.dart

```diff
@@ -104,8 +103,8 @@ class CancelableFsFetch {
     EntityInfo b, {
     bool isDirectory = false,
   }) {
-    EntityInfo item1 = a;
-    EntityInfo item2 = b;
```

Comment:
would keep the type here, too js looking else, don't like the readability it loses